doc/user-guide/src/concepts/toolchains.md

```diff
@@ -15,7 +15,9 @@ Standard release channel toolchain names have the following form:
 ```
 <channel>[-<date>][-<host>]
 
-<channel>       = stable|beta|nightly|<major.minor>|<major.minor.patch>
+<channel>       = stable|beta|nightly|<versioned>[-<beta tag>]
+<versioned>     = <major.minor>|<major.minor.patch>
+<beta tag>      = beta[.<number>]
```

Comment:
Let's call this `prerelease` to align with the terminology used in the codebase.

```suggestion
<prerelease>      = beta[.<number>]
```